tests/integration/conftest.py

```diff
@@ -25,6 +25,18 @@
     Path(__file__).parent / "data" / "LC08_L1TP_090084_20160121_20200907_02_T1"
 )
 
```

Comment:
I'd rather keep all of this (the fixtures etc) in `test_prepare_landsat_l1.py`, as they're only used within that file. `conftest.py` is for globals to share across multiple test files and is already quite large.

(Feel free to ditch the extra "expect_*" function complexity too. And maybe inline the expected dictionaries completely into the tests without fixtures. That extra complexity existed because other tests are built with multiple packaging types and had to be constructed multiple times differently. But yours have just has one input and one output per test.)

Reply:
I have tidied the test cases, see what you think.